### PR TITLE
Adds asset registration guide

### PR DESCRIPTION
**Pull request by augustocollado**
### Description

This PR adds the foreign asset registration guide.
It also creates a snippet for shared text, and reorders images from the xcm channel management article.

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: builders/manage/dapp/xcm-channels.md [hunk 1]
```diff
@@ -17,15 +17,7 @@ In this guide, you'll learn how to use the [Tanssi dApp](https://apps.tanssi.net
...
-The Tanssi dApp makes it easy for the appchain governor to manage cross-chain communication channels. It allows requesting new channels, accepting incoming channel opening requests, closing existing channels, and more. To access the channel management panel, head to the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}, and then: 
-
-1. Click on the **Manage** button
-2. Click on the **XCM** button.
-
-![Accessing the channel management panel](/images/builders/manage/dapp/xcm-channels/xcm-channels-1.webp)
-
-!!! note
-    If you don't see your appchain's details on the dashboard, make sure to comply with the [prerequisites](#checking-prerequisites).
+--8<-- 'text/builders/manage/dapp/cross-chain-config-panel.md'
```
Comment (eshaben) [inline builders/manage/dapp/xcm-channels.md:20]: Maybe I'm missing it, but where else is this used?

File: builders/manage/dapp/register-foreign-assets.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+--8<-- 'text/builders/manage/dapp/prerequisites.md'
+
+## Accessing the Foreign Assets Registration Panel {: #accesing-foreign-assets-management-panel }
+
+--8<-- 'text/builders/manage/dapp/cross-chain-config-panel.md'
+
+The panel will show your appchain's cross-chain configuration along with several available actions. Regarding foreign assets, the relevant elements you are presented with are:
+
+1. **Registered Assets panel** - this section will group and present to you all registered assets your appchain already has available
+2. **Assets List** - already registered foreign assets will be displayed in this section, along with their associated information such as asset name, symbol, ID, total supply, and appchain ID where it is native
```
Comment (eshaben) [inline builders/manage/dapp/register-foreign-assets.md:26]: I'd maybe rename this as "Registered Assets list" ? but not a big deal either way

**Review by eshaben — COMMENTED** [review]
Just want to check - "Foreign assets" is the term to use?

File: builders/manage/dapp/register-foreign-assets.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+title: Register Foreign Assets
```
Comment (albertov19) [inline builders/manage/dapp/register-foreign-assets.md:2]: We don't mention anything in regards to the minimum runtime version right?

**Review by albertov19 — APPROVED** [review]
LGTM !

File: builders/manage/dapp/xcm-channels.md (hunk 1 above)
Comment (augustocollado) [reply builders/manage/dapp/xcm-channels.md:20]: It's also used in the manage XCM channels guide

**Comment by augustocollado** [comment]
> Just want to check - "Foreign assets" is the term to use?

Can't think of a better name :)

**Comment by albertov19** [comment]
> > Just want to check - "Foreign assets" is the term to use?
> 
> Can't think of a better name :)

I would agree with Erin - I think foreign is a very Substrate terminology. In the past we thought of  "External" as a friendlier version of "Foreign

**Review by albertov19 — DISMISSED** [review]
LGTM!

File: .snippets/text/builders/manage/dapp/cross-chain-config-panel.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+The Tanssi dApp makes it easy for the appchain governor to manage cross-chain communication configurations. It allows requesting new channels, accepting incoming channel opening requests, closing existing channels, managing external assets, and more. To access the cross-chain management panel, head to the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}, and then: 
+
+1. Click on the **Manage** button
+2. Click on the **XCM** button.
```
Comment (eshaben) [inline .snippets/text/builders/manage/dapp/cross-chain-config-panel.md:4]:
```suggestion
2. Click on the **XCM** button
```

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**